modules/ROOT/attachments/admin-api.yaml

```diff
@@ -809,6 +809,145 @@ paths:
         200:
           description: Stop maintenance success
           content: {}
+  /v1/migrations:
+    get: 
+      summary: List all available migrations
+      operationId: list_migrations
+      responses: 
+        200:
+          description: OK
+          content: 
```

Comment:
What is the schema for the response here? I don't see one in the spec https://github.com/redpanda-data/redpanda/blob/dev/src/v/redpanda/admin/api-doc/migration.json#L8 and in the [docs preview](https://deploy-preview-798--redpanda-docs-preview.netlify.app/api/admin-api/#get-/v1/migrations) it displays _Schema not found_.

Reply:
List of items, each being either an `outbound_migration_state` or an `inbound_migration_state`. I'll need to add it.